signuplogin.html

```diff
@@ -18,6 +18,20 @@
 </head>
 
 <body>
+
+<header>
+		<nav>
+			<div>
+				<input type="checkbox" class="checkbox" id="chk" />
+				<label class="label" for="chk">
+				  <i class="fas fa-moon"></i>
+				  <i class="fas fa-sun"></i>
+				  <div class="ball"></div>
+				</label>
+			  </div>
+	
+	</nav>
```

Comment:
Something is wrong with the code of toggle button. The changes are not reflected and hence the button is not visible.